web/src/containers/BudgetSummary.js

```diff
@@ -146,28 +138,17 @@ const BudgetSummary = ({ activities, data, years }) => (
     ))}
 
     <table className="budget-table">
-      <caption className="ds-h4">
-        Project Activities Totals
-      </caption>
+      <caption className="ds-h4">Project Activities Totals</caption>
       <thead>
         <tr>
-          <th id="summary-budget-null1" />
```

Comment:
This one actually changed. All that other stuff is just prettier reformatting lines. :pixar-joy: